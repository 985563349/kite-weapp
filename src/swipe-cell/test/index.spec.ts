import path from 'node:path';
import simulate from 'miniprogram-simulate';

describe('swipe-cell', () => {
  const KSwipeCell = simulate.load(path.resolve(__dirname, '../swipe-cell'), 'k-swipe-cell', {
    rootPath: path.resolve(__dirname, '../../'),
  });

  test('should render swipe-cell and match snapshot', () => {
    const comp = simulate.render(KSwipeCell);
    comp.attach(document.createElement('parent-wrapper'));

    expect(comp.toJSON()).toMatchSnapshot();
  });
});
