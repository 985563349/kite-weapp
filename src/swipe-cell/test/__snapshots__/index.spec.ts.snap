// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`swipe-cell should render swipe-cell and match snapshot 1`] = `
<wx-view
  $gdc="{{[Function]}}"
  $wxs:bind:touchcancel="{{[Function]}}"
  $wxs:bind:touchend="{{[Function]}}"
  $wxs:bind:touchstart="{{[Function]}}"
  $wxs:capture-bind:touchmove="{{[Function]}}"
  $wxs:change:disabled="{{[Function]}}"
  $wxs:change:opened="{{[Function]}}"
  class="k-swipe-cell k-class"
  disabled="{{false}}"
  id=""
  opened="{{null}}"
  style=""
  bind:tap="onTap"
  catch:touchmove="noop"
>
  <wx-view
    class="k-swipe-cell-content"
  >
    <wx-view
      class="k-swipe-cell-left"
    />
    <wx-view
      class="k-swipe-cell-right"
    />
  </wx-view>
</wx-view>
`;
